options/options.go

```diff
@@ -240,3 +247,21 @@ func validateFormat(format string) bool {
 		return false
 	}
 }
+
+// isCheckValidForLocalRepo returns an error if the check is not valid for local repository.
+func (o *Options) isCheckValidForLocalRepo() error {
+	// This is a remote repository
+	if o.Local == "" {
+		return nil
+	}
```

Comment:
Use this `if` on the caller side instead? Something like:

```
if o.Local != "" {
  if err := os.isValidForLocalRepo(); err != nil {
    ...
  }
}
```

Reply:
I don't think this is applicable any more.

Reply:
Resolving this, can use Laurent's suggestion.